aclopez/SinergiaEquipo/readme.md

```diff
@@ -0,0 +1,35 @@
+
+Quizá si una máquina te dice que tienes una vulnerabilidad porque estás paginando los resultados de tu consulta de EF no te moleste tanto si te lo dice tu compañero de enfrente. Claro, tu compañero de enfrente, ese con el que "compites" por tu puesto. A Resharper nadie le va a quitar el puesto... 
+
+Yo he estado en los dos lados. En el de Juan y en el de su compañero. Me da pena que vivamos en un mundo capitalista donde se "compite" por un puesto de trabajo y muchas veces los sueldos de los programadores no sean acordes a su función en el equipo, pero son las reglas del tiempo que nos ha tocado vivir y eso debería quedar aparte. Nunca se compite con tus compañeros de equipo. Esa lucha del sueldo será siempre entre tu jefe y tú, pero tu equipo debe quedar fuera de esa batalla. Por tanto, nadie compite contra nadie, mucho menos dentro de un mismo equipo creando un producto software, por mucho que esté en juego el puesto de jefe del producto estrella de tu empresa donde te da acceso directo a cosas de jefes, eso será más política que desarrollo de software, y si no eres capaz de tener un equipo sin ese tipo de problemas, tu usuarios finales "oiran" esas discursiones en tu producto final.
+
+Evidentemente influye la manera que tenemos de expresarnos. Es algo que no hay que esconder. Pero vuelvo a la pregunta de: ¿es Juan más violento que Resharper? Es muy complicado ser siempre complaciente con tu receptor, sobretodo si ya te han etiquetado de "hater". Confiamos ciegamente en lo que dice Resharper, porque sin lugar a dudas es mucho más fiable que cualquier persona, pero no somos capaces de preguntar después de una crítica de un compañero el porqué de su feedback negativo. Es decir, quizás es cierto que sin eso o no funcione, pero cuando alguien dice "eso es una mierda" hay un motivo detrás, y si lo único que pensamos es en nuestro ego y en que ha dicho que lo nuestro es una mierda por atacarnos, nos vamos a perder ese porqué, esa oportunidad de aprender algo nuevo. Ya que seamos realistas, es una oportunidad. Puede que eso que esconde ese "esto es una mierda", "no sé como programáis aquí" o un largo etcétera de expresiones "violentas" no sea más que un desconocimiento por parte del emisor, pero ¿no merece la pena escucharlo si tenemos alguna duda al respecto?
```

Comment:
sobretodo -> sobre todo
sé como programáis -> sé cómo programáis

Reply:
sobre todo

Reply:
porque, sin lugar a dudas,...

Reply:
Es decir, quizás sea cierto que sin eso no funcione...

Reply:
Ya que, seamos realistas,...